utils/selection.py

```diff
@@ -0,0 +1,588 @@
+        for i in range(5):
+            if i < len(current_choices):
+                global_index = i + 1 + current_page * 10
+                button = disnake.ui.Button(
+                    label=str(global_index),
+                    style=disnake.ButtonStyle.secondary,
+                    disabled=self.expired,
+                    custom_id=f"select_{global_index}",
+                )
+            else:
+                # Placeholder button when insufficient choices
+                button = disnake.ui.Button(
+                    label=str(i + 1 + current_page * 10),
+                    style=disnake.ButtonStyle.secondary,
+                    disabled=True,
+                    custom_id=f"placeholder_{i}",
+                )
+            self.add_item(button)
+
+        # Row 2: Selection buttons 6-10 (fill with placeholders if needed)
+        for i in range(5, 10):
+            if i < len(current_choices):
+                global_index = i + 1 + current_page * 10
+                button = disnake.ui.Button(
+                    label=str(global_index),
+                    style=disnake.ButtonStyle.secondary,
+                    disabled=self.expired,
+                    custom_id=f"select_{global_index}",
+                )
+            else:
+                # Placeholder button when insufficient choices
+                button = disnake.ui.Button(
+                    label=str(i + 1 + current_page * 10),
+                    style=disnake.ButtonStyle.secondary,
+                    disabled=True,
+                    custom_id=f"placeholder_{i}",
+                )
+            self.add_item(button)
+
+        # Row 3: Navigation buttons (only show if more than 10 total results)
+        if len(choices) > 10:
+            # Previous button
+            prev_button = disnake.ui.Button(
+                label="◀ Previous",
+                style=disnake.ButtonStyle.secondary,
+                disabled=(current_page == 0 or self.expired),
+                custom_id="prev",
+            )
+            self.add_item(prev_button)
+
+            # Next button
+            next_button = disnake.ui.Button(
+                label="Next ▶",
+                style=disnake.ButtonStyle.secondary,
+                disabled=(current_page >= total_pages - 1 or self.expired),
+                custom_id="next",
+            )
+            self.add_item(next_button)
+
+        # Cancel button - clean styling
+        cancel_button = disnake.ui.Button(
+            label="Cancel", style=disnake.ButtonStyle.danger, disabled=self.expired, custom_id="cancel"
+        )
+        self.add_item(cancel_button)
+
+    async def interaction_check(self, interaction: disnake.MessageInteraction) -> bool:
+        """Handle interactions on expired views (just in case)"""
+        if self.expired:
+            await interaction.response.send_message("⏰ This selection menu has expired.", ephemeral=True)
+            return False
```

Comment:
Is there a chance this gets called? The expiration should disable the buttons, right?